components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -967,36 +956,6 @@ then any unregistered scopes(excluding internal scopes
         return allowedOAuthScopes;
     }
 
-    private static void setConsentRequiredScopesToOAuthParams(AuthenticatedUser user, OAuth2Parameters params)
-            throws OAuthSystemException {
-
-        try {
-            //Filter out OIDC scopes and unregistered scopes to prevent those scopes prompt for consent in the consent
-            // page.
-            List<String> consentRequiredScopes = dropOIDCAndUnregisteredScopesFromConsentRequiredScopes(params);
```

Comment:
Where is dropping unregistered scopes happening now with the proposed changes? We may need to call that part seperately.